nilearn/input_data/nifti_labels_masker.py

```diff
@@ -0,0 +1,303 @@
+"""
+Transformer for computing ROI signals.
+"""
+
+import numpy as np
+
+from sklearn.base import BaseEstimator, TransformerMixin
+from sklearn.externals.joblib import Memory
+
+import nibabel
+
+from .. import _utils
+from .._utils import logger
+from .._utils import CacheMixin
+from .._utils.niimg_conversions import _check_same_fov
+from .. import signal
+from .. import region
+from .. import masking
+from .. import image
+
+
+def _compose_err_msg(msg, **kwargs):
```

Comment:
I know, there is a similar situation with a helper function in the tests. What's your suggestion to handle this?